pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,41 @@
+{lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, SDL, SDL_mixer, SDL_ttf}:
+
+stdenv.mkDerivation rec {
+   pname = "hex-a-hop";
+   version = "1.1.0";
+
+   src = fetchurl {
+     url = "https://downloads.sourceforge.net/project/hexahop/1.1.0/hex-a-hop-1.1.0.tar.gz";
+     sha256 = "sha256-gL8Z2TaoQwyrW8Ro7mgnw4BS4CbGCet+zhYxfnkyoNY=";
+   };
+
+   buildInputs = [ SDL SDL_mixer SDL_ttf copyDesktopItems];
+
+   makeFlags = [ "PREFIX=$(out)" ];
+
+   desktopItems = makeDesktopItem rec {
+     name = pname;
+     exec = name;
+     icon = name;
+     desktopName = name;
+     comment = meta.description;
+     type = "Application";
+     categories = [ "Game" "LogicGame" ];
+     startupNotify = false;
+   };
+
+   postFixup = ''
+     install -Dm644 ${./hex-a-hop.png} $out/share/icons/${pname}.png
+   '';
+
+   meta = {
+     description = "A puzzle game based on hexagonal tiles";
+     homepage = "http://hexahop.sourceforge.net";
+     license = lib.licenses.gpl2;
```

Comment:
-Only or -Plus?

Reply:
On debian it's tagged as GPL2+ https://metadata.ftp-master.debian.org/changelogs/main/h/hex-a-hop/stable_copyright

Reply:
changed it to gpl2Plus in a0062efb737846f9d62f646faf3107976cf6291b